### PR TITLE
Export Locations module for correct Location namespacing

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/nodejs/location.ts [hunk 1]
```diff
@@ -12,38 +12,40 @@
...
-export let UKSouth:            Location = "uksouth";
-export let UKWest:             Location = "ukwest";
-export let WestCentralUS:      Location = "westcentralus";
-export let KoreaCentral:       Location = "koreacentral";
-export let KoreaSouth:         Location = "koreasouth";
-export let FranceCentral:      Location = "francecentral";
-export let FranceSouth:        Location = "francesouth";
-export let SouthAfricaNorth:   Location = "southafricanorth";
-export let SouthAfricaWest:    Location = "southafricawest";
+export module Locations {
```
Comment (mikhailshilkov) [inline sdk/nodejs/location.ts]: Why capital `L`?

Comment (stack72) [reply sdk/nodejs/location.ts]: To go with the convention we have used elsewhere in other providers - @jen20 thoughts here?

Comment (stack72) [reply sdk/nodejs/location.ts]: @mikhailshilkov this comes from pulumi-aws#599 btw - I just spoke to @jen20 about it

Comment (jen20) [reply sdk/nodejs/location.ts]: The pattern comes from pulumi/pulumi-aws#599 - probably worth sticking with at this point?

**Pull request merged**